### ISSUE TITLE
Task 8 input nodes

### ISSUE THREAD

**Issue by kareemostafa** [chulminy/CIVE497-CIVE700#83]
can we use something other than X1 and X2 in our input nodes (e.g. X1^2) ?

**Comment by chulminy** [comment]
Yes. You can use. However, I could solve without introducing the other nodes.
